frontend/src/embedding-sdk-bundle/components/public/dashboard/tests/setup.tsx

```diff
@@ -184,6 +186,10 @@ export const setupSdkDashboard = async ({
 
   const user = createMockUser();
 
+  if (hasEmbeddingEnterprisePlugin) {
+    setupEnterpriseOnlyPlugin("embedding");
+  }
```

Comment:
nit: Is there a reason for the plugin activation to move above `state` initialization? If not I'd like to see it stay in the same order.